inertia/tests/test_rendering.py

```diff
@@ -109,3 +109,22 @@ def test_that_csrf_is_included_even_on_initial_page_load(self):
     response = self.client.get('/props/')
 
     self.assertIsNotNone(response.cookies.get('csrftoken'))
+
+class DeferredPropsTestCase(InertiaTestCase):
```

Comment:
Great test cases for the base case! Could we also add two additional tests in this TestCase to verify that the props are returned correctly when they're requested via the deferred request? I think they will end up looking something like the `LazyPropsTestCase#test_lazy_props_are_included_when_requested` test above.